index.js

```diff
@@ -84,16 +109,25 @@ export default {
 			}
 			this.stopScroll();
 		} else {
-			let editor;
-			if (e.button === 1 && (editor = e.target.closest("atom-text-editor:not([mini])")) && this.editor !== editor) {
+			const pane = e.target.closest(
+				"atom-text-editor:not([mini]), " +
+				".results-view-container, " +
+				".settings-view .panels-item, " +
+				".tree-view"
```

Comment:
Could we make this easier to add to?

Maybe something like:
```js
const selectors = [
	"atom-text-editor:not([mini])", // editors
	".results-view-container", // project find results
	".settings-view .panels-item", // settings panel
	".settings-view .config-menu", // settings menu
	".tree-view", // tree view
];
const pane = e.target.closest(selectors.join(", "));
```

also add `".settings-view .config-menu"`